### ISSUE TITLE
Export to Curl, Python, Raw fails silently.

### ISSUE THREAD

**Issue by ffernand** [mitmproxy/mitmproxy#965]
Using the new Export options made available on mitmproxy 0.16 doesn't seem to work.  It goes as far as providing the option to export either as a curl command, python code, or raw, but after the selection is made the prompt disappears.

No errors are evident and looking to the file system in the folder where mitmproxy was started doesn't show any new files where an exported command may have written to.  Exiting mitmproxy doesn't show any errors or a stack trace.

##### Steps to reproduce the problem:

1. Start mitmproxy on the command line.
2. Run the following curl command in a different terminal `curl --proxy localhost:8080 http://www.google.com`
3. Back to the mitmproxy terminal, select the intercepted url and press `E`
4. The prompt `Export (as curl command,as python code,as raw request)?` shows up and pressing the letter `c`, `p`, or `r` does nothing.

##### What is the expected behavior?

Unsure; the feature is new and I haven't see it work correctly.

##### What went wrong?

No opportunity to see the exported curl command.

##### Any other comments?

```
$ uname -a
Linux personal 3.13.0-77-generic #121-Ubuntu SMP Wed Jan 20 10:50:42 UTC 2016 x86_64 x86_64 x86_64 GNU/Linux

$ lsb_release -a
No LSB modules are available.
Distributor ID: Ubuntu
Description:    Ubuntu 14.04.4 LTS
Release:    14.04
Codename:   trusty
```
---

Mitmproxy Version: 0.16
Operating System: Linux

**Comment by mhils** [comment]
cc @dufferzafar

**Comment by dufferzafar** [comment]
@ffernand Thanks for a well written issue! 

I think you expect that the Export feature will save the flow to some file on disk, but that is not the case! Here 'Export' just means 'Copy to clipboard'. 

So, try exporting and then do a Ctrl + V somewhere. See if it works?

@mhils While I was working on this, I too felt that the wording was a bit unclear. since mitmproxy already has the `P` key to copy flow contents to clipboard. Your thoughts?

**Comment by ffernand** [comment]
Thanks very much for the fast response!

Looking at my clipboard history, I do indeed see all the exports I've made.  And the feature works great!  As a suggestion... maybe if the prompt said `Export to clipboard (...` instead of just `Export (...` ?

Thanks very much for this feature... I know I'll be using this routinely :)

**Comment by mhils** [comment]
We're discussing ways to fix this in #976. :smiley:
